tests/strip/utils.toit

```diff
@@ -25,17 +27,24 @@ backticks-failing args/List -> string:
   process.stdin.close
 
   // We are merging stdout and stderr into one stream.
+  stdout-done := monitor.Latch
```

Comment:
An alternative would be a single semaphore.

Reply:
Agreed. I thought of the same thing.
But I had already started with the latches, and they read nicer.